src/ai/.x/templates/openai-asst-webpage-js/src/OpenAIEnvInfo.js

```diff
@@ -8,26 +8,25 @@
 <#@ parameter type="System.String" name="AZURE_OPENAI_SYSTEM_PROMPT" #>
 <#@ parameter type="System.String" name="OPENAI_API_KEY" #>
 <#@ parameter type="System.String" name="OPENAI_MODEL_NAME" #>
-class OpenAIEnvInfo {
-
+export class OpenAIEnvInfo {
   // NOTE: Never deploy your key in client-side environments like browsers or mobile apps
   //  SEE: https://help.openai.com/en/articles/5112595-best-practices-for-api-key-safety
 
-  static ASSISTANT_ID = process.env.ASSISTANT_ID ?? "<#= ASSISTANT_ID #>";
+  static ASSISTANT_ID = import.meta.env.ASSISTANT_ID ?? "<insert your OpenAI assistant ID here>";
 
-  static AZURE_CLIENT_ID = process.env.AZURE_CLIENT_ID ?? null;
-  static AZURE_TENANT_ID = process.env.AZURE_TENANT_ID ?? null;
+  static AZURE_CLIENT_ID = import.meta.env.AZURE_CLIENT_ID ?? null;
+  static AZURE_TENANT_ID = import.meta.env.AZURE_TENANT_ID ?? null;
 
-  static AZURE_OPENAI_API_KEY = process.env.AZURE_OPENAI_API_KEY ?? "<#= AZURE_OPENAI_API_KEY #>";
-  static AZURE_OPENAI_API_VERSION = process.env.AZURE_OPENAI_API_VERSION ?? "<#= AZURE_OPENAI_API_VERSION #>";
-  static AZURE_OPENAI_ENDPOINT = process.env.AZURE_OPENAI_ENDPOINT ?? "<#= AZURE_OPENAI_ENDPOINT #>";
-  static AZURE_OPENAI_CHAT_DEPLOYMENT = process.env.AZURE_OPENAI_CHAT_DEPLOYMENT ?? "<#= AZURE_OPENAI_CHAT_DEPLOYMENT #>";
+  static AZURE_OPENAI_API_KEY = import.meta.env.AZURE_OPENAI_API_KEY ?? "<insert your Azure OpenAI API key here>";
+  static AZURE_OPENAI_API_VERSION = import.meta.env.AZURE_OPENAI_API_VERSION ?? "<insert your Azure OpenAI API version here>";
+  static AZURE_OPENAI_ENDPOINT = import.meta.env.AZURE_OPENAI_ENDPOINT ?? "<insert your Azure OpenAI endpoint here>";
+  static AZURE_OPENAI_CHAT_DEPLOYMENT = import.meta.env.AZURE_OPENAI_CHAT_DEPLOYMENT ?? "<insert your Azure OpenAI chat deployment name here>";
 
-  static AZURE_OPENAI_SYSTEM_PROMPT = process.env.AZURE_OPENAI_SYSTEM_PROMPT ?? "<#= AZURE_OPENAI_SYSTEM_PROMPT #>";
+  static AZURE_OPENAI_SYSTEM_PROMPT = import.meta.env.AZURE_OPENAI_SYSTEM_PROMPT ?? "You are a helpful AI assistant.";
```

Comment:
all the other variables fail if not set, just double checking that this is what we want as a default here. Not sure what a "failure" system prompt would be unless we set it to something like "no matter what I say respond with 'you have not set your system prompt'"

Reply:
I like the idea of propagating the error message to the user's UI so they can get instant feedback (vs having to switch back to terminal console or open browser console).

Reply:
Oh I just noticed that this is currently the case. Missing env vars are error message is shown to users in the UI. So ignore my previous comment :)